docs/README_sp.md

```diff
@@ -0,0 +1,195 @@
+![npm-audit-periodic](https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/workflows/npm-audit-periodic/badge.svg)
+
+<p align="center">
+  <span>Español</span> |
+  <a href="../README.md">English</a> |
+</p>
+
+# Aplicación Web de referencia para Cloud Run for Anthos
+
+Este repositorio, que incluye todos los flujos de trabajo y automatizaciones asociados, representa un conjunto de mejores prácticas dirigidas a demostrar una arquitectura de referencia para crear una aplicación web en Google Cloud utilizando Cloud Run para Anthos.
+
+Se puede encontrar una descripción detallada de la arquitectura de la aplicación web en [architecture.md][].
+
+## Prerequisitos
+
+### Ambiente de Desarrollo
+
+*NOTA: los pasos de esta guía asumen que se está trabajando en un ambiente de desarrollo basado en POSIX.
+development environment.*
+
+El único requerimiento para ejecutar este ejemplo como se muestra en este repositorio es una instalación funcional de `gcloud`. Opcionalmente, tener `make` instalado lo cual le permitirá hacer uso de los objetivos de conveniencia provistos en el [`makefile`][].
+
+
+*NOTA: Su cuenta de usuario de `gcloud` debe tener [Owner permission][] sobre el proyecto para poder completar la configuración de la aplicación.*
```

Comment:
Is there an appropriate translation available for "Owner permission" or does it make more sense to leave it as-is?

Reply:
I think "permiso de propietario" would make sense given the terms used [here](https://cloud.google.com/iam/docs/understanding-roles?hl=es#primitive_role_definitions)